cmd/boulder-ra/main.go

```diff
@@ -31,11 +31,12 @@ import (
 
 type Config struct {
 	RA struct {
+		// RateLimitPoliciesFilename is deprecated.
```

Comment:
In general we don't move struct fields to the top when we deprecate them -- we only do that in features.go because we frequently have many deprecated feature flags and it's nice to put them all in one spot. For a more generic deprecation like this, I'd leave the field in place so that things like `git blame` work better.